bcipy/display/rsvp/rsvp_disp.py

```diff
@@ -203,6 +204,16 @@ def do_sequence(self):
                 sti_label = self.sti.text
             else:
                 self.sti.image = self.stim_sequence[idx]
+                
+                #Retrieve image width and height
+                with Image.open(self.sti.image) as pillow_image:
+                    image_width, image_height = pillow_image.size
+                #Resize image so that its largest dimension is the stimuli size defined in the parameters file
+                if image_width >= image_height:
+                    self.sti.size = (self.height_stim, (image_height/image_width) * self.height_stim)
```

Comment:
syntax error. should be `(self.height_stim, (image_height / image_width) * self.height_stim)`